Doc/whatsnew/3.14.rst

```diff
@@ -1081,6 +1081,14 @@ The behavior of :func:`!gc.collect` changes slightly:
 
 (Contributed by Mark Shannon in :gh:`108362`.)
 
+Platform Support
+================
+
+* :pep:`776`: Emscripten is now an officially supported platform at
+  :pep:`tier 3 <11#tier-3>`. As a part of this effort, more than 25 bugs in
+  Emscripten libc were fixed :gh:`127146`. Emscripten now includes support
+  for ``ctypes`` (:gh:`127683`), ``termios``, and ``fcntl``, as well as
+  experimental support for PyREPL (:gh:`136931`).
```

Comment:
It might be good to link to https://docs.python.org/3/tutorial/appendix.html#tut-interac when discussing PyREPL. I think it is still new enough that people may not know what PyREPL is vs the the REPL in general.